WordPress/WordPressTest/MediaServiceRemoteRESTTests.swift

```diff
@@ -0,0 +1,144 @@
+import XCTest
+@testable import WordPress
+
+class MediaServiceRemoteRESTTests: XCTestCase {
+
+    let mockRemoteApi = MockWordPressComRestApi()
+    var mediaServiceRemote: MediaServiceRemoteREST!
```

Comment:
I believe `MediaServiceRemoteREST` doesn't hold any state, so you can just define it here once instead of on `setUp`. In any case, I'd initialize here and lose the `!`